ql/src/java/org/apache/hadoop/hive/ql/plan/HiveOperation.java

```diff
@@ -120,6 +120,8 @@
       new Privilege[]{Privilege.DROP}),
   ALTER_MATERIALIZED_VIEW_REWRITE("ALTER_MATERIALIZED_VIEW_REWRITE", HiveParser.TOK_ALTER_MATERIALIZED_VIEW_REWRITE,
       new Privilege[]{Privilege.ALTER_METADATA}, null),
+  ALTER_MATERIALIZED_VIEW_REBUILD("ALTER_MATERIALIZED_VIEW_REBUILD", HiveParser.TOK_ALTER_MATERIALIZED_VIEW_REBUILD,
```

Comment:
Why do we need to add this operation? I saw a comment about it in another conversation but it was not clear over there.

In the q file below, I see that that this change is leading to inconsistent operation type when rebuild is executed (QUERY) vs skipped (ALTER_MATERIALIZED_VIEW_REBUILD). This should not happen: The operation type should be the same in both cases.

Reply:
Without this an exception is thrown at execution time.
```
java.lang.IllegalStateException: Unknown HiveOperation(null) for queryId=krisz_20210104002338_38173ed5-af09-4e65-af17-8be1dc49e62e
	at org.apache.hadoop.hive.ql.lockmgr.DbTxnManager.verifyState(DbTxnManager.java:315)
	at org.apache.hadoop.hive.ql.lockmgr.DbTxnManager.acquireLocks(DbTxnManager.java:389)
...
	at org.apache.hadoop.hive.ql.Driver.lockAndRespond(Driver.java:329)
	at org.apache.hadoop.hive.ql.Driver.runInternal(Driver.java:193)
...
```

Fixed the inconsistency by setting back the operation type to `ALTER_MATERIALIZED_VIEW_REBUILD` after analyzing the query like in the case of `CREATE_MATERIALIZED_VIEW`